LocalSettings.php

```diff
@@ -2862,7 +2866,6 @@
 		'pfsolutionswiki' => true,
 		'pgnwikiwiki' => true,
 		'plasticssongcontestwiki' => true,
-		'plazmaburstwiki' => true,
```

Comment:
@Wiki1776 This edit removes your last one. Is this intentional?

Reply:
no, it's a mistake I'm going to fix.